app.js

```diff
@@ -44,21 +44,31 @@ var exitnodes = ['45.34.140.42'];
 var isAlive = function(req, res) {
   var numberOfGateways = req.query.numberOfGateways || 'n/a';
   var numberOfRoutes = req.query.numberOfRoutes || 'n/a';
-  mjs.set('alive', 'Yes! Connecting [' + numberOfRoutes + '] nodes via [' + numberOfGateways + '] gateways.', {expires:120}, function(err) {
+
+  var handleErr = function(err) {
     if (err) {
+
       console.log("Error setting key: " + err);
+
       res.render('error', {
         message: err.message,
         error: err
       });
     } 
-  });
+  };
+
+  var msg = `Yes! Connecting [${numberOfRoutes}] nodes via [${numberOfGateways}] gateways.`
+  mjs.set('alive', msg, {expires:120}, handleErr);
+  var jsonResults = {numberOfGateways: numberOfGateways, numberOfRoutes: numberOfRoutes};
+  mjs.set('alivejson', JSON.stringify(jsonResults), {expires: 120}, handleErr);
```

Comment:
Hmm. Would it make sense to only store the json representation? And render the string template in the get / handler?

Reply:
Fixed in 5bd77db